sdk/key_vault/src/key.rs

```diff
@@ -296,6 +344,52 @@ impl<'a, T: TokenCredential> KeyClient<'a, T> {
         result.algorithm = algorithm;
         Ok(result)
     }
+
+    /// Decrypt a single block of encrypted data.
+    /// The DECRYPT operation decrypts a well-formed block of ciphertext using the target encryption key and specified algorithm.
+    /// This operation is the reverse of the ENCRYPT operation; only a single block of data may be decrypted, the size of this block is dependent on the target key and the algorithm to be used.
+    /// The DECRYPT operation applies to asymmetric and symmetric keys stored in Vault or HSM since it uses the private portion of the key. This operation requires the keys/decrypt permission.
+    pub async fn decrypt(
+        &mut self,
+        algorithm: EncryptionAlgorithm,
+        key_name: &str,
+        key_version: &str,
```

Comment:
Even in `DecryptParameters`, `version` (not `key_version`) is optional despite what the swagger says (path variables can't be optional, but they handle this correctly on the server.